tests/integration/example/environment/appOnlyEnv/app_only_env.go

```diff
@@ -35,27 +37,32 @@ func NewAppOnlyEnv(id string) *AppOnlyEnv {
 	}
 }
 
-// GetName return environment ID
+// GetName implement the function in TestEnv return environment ID
 func (appOnlyEnv *AppOnlyEnv) GetName() string {
 	return appOnlyEnv.EnvID
 }
 
-// Bringup doing setup for AppOnlyEnv
+// Bringup implement the function in TestEnv
+// Create local temp dir. Can be manually overrided if necessary.
 func (appOnlyEnv *AppOnlyEnv) Bringup() (err error) {
```

Comment:
Can we have consistent names for start up and shut down between Env and Comp (or even Process)?